### PR TITLE
Pull #990: Define variable for non-main-files-suppressions file

### PR DESCRIPTION
**Pull request by stoyanK7**
From https://github.com/checkstyle/checkstyle/pull/12755#discussion_r1112055478
> So before mergin this PR we need 1 PR for checkstyle repo to make it as variable and 1 PR for sevnu to define this variable.

Related to https://github.com/checkstyle/checkstyle/pull/12761

### CODE REVIEW

**Comment by rnveach** [comment]
@stoyanK7 Are you still working on the PR in main repo? I need to see it with this to verify the whole flow will work.

**Comment by stoyanK7** [comment]
@rnveach I will come back to it later today. I suppose you want me to connect it to this PR and see green CI.

**Comment by rnveach** [comment]
If you can make main repo CI green and show proof it is invoking the new stuff, then that will be even better as it will show it actually works.

**Comment by stoyanK7** [comment]
@rnveach To prove that this PR works, I think we actually need to change `pom.xml` here and that's it:
```xml
<checkstyle.nonMain.configLocation>https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-${checkstyle.version}/config/checkstyle_non_main_files_checks.xml</checkstyle.nonMain.configLocation>
```
to
```xml
<checkstyle.nonMain.configLocation>https://raw.githubusercontent.com/checkstyle/checkstyle/master/config/checkstyle_non_main_files_checks.xml</checkstyle.nonMain.configLocation>
```

**Comment by rnveach** [comment]
https://github.com/checkstyle/checkstyle/commit/40f409271dd0f6e387bd0be14313589e52c14881 is what I was looking for. It was merged without my review so i didn't realize.

Truthfully, this PR should have been made first, and then a PR in main repo referencing this and adding extra commit to test and show it is hooking in.

**Comment by romani** [comment]
I am not sure what I did wrong. What blocks this PR to be merged?

**Comment by stoyanK7** [comment]
I think nothing?

File: sevntu-checks/pom.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@
     <checkstyle.version>10.7.0</checkstyle.version>
     <checkstyle.configLocation>https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-${checkstyle.version}/config/checkstyle_checks.xml</checkstyle.configLocation>
     <checkstyle.nonMain.configLocation>https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-${checkstyle.version}/config/checkstyle_non_main_files_checks.xml</checkstyle.nonMain.configLocation>
+    <checkstyle.non-main-files-suppressions.file>config/checkstyle_non_main_files_suppressions.xml</checkstyle.non-main-files-suppressions.file>
```
[the remark this answers is not in the document]
Comment (rnveach) [reply sevntu-checks/pom.xml:24]: @romani I'll leave this to you.

[the remark this answers is not in the document]
Comment (romani) [reply sevntu-checks/pom.xml:24]: Let's use for properties camelcase

[the remark this answers is not in the document]
Comment (romani) [reply sevntu-checks/pom.xml:24]: But see how badly inconsistent we are on property names... .
Let's do this in separate PR.

**Comment by rnveach** [comment]
@stoyanK7 It seems there is issues with the travis runs and it is actually failing. This PR just had a 10 minute time out run and it was not confirmed to be working. Another PR is failing.

https://app.travis-ci.com/github/sevntu-checkstyle/sevntu.checkstyle/jobs/597291051
> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.2.1:check (default) on project sevntu-checkstyle-sonar-plugin: Failed during checkstyle execution: Unable to find configuration file at location: https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-10.8.0/config/checkstyle_checks.xml: Could not find resource 'https://raw.githubusercontent.com/checkstyle/checkstyle/checkstyle-10.8.0/config/checkstyle_checks.xml'. -> [Help 1]

**Comment by stoyanK7** [comment]
@rnveach On it. That was expected after the release happened.

**Pull request merged**